### ISSUE TITLE
IdentityServer3.AspNetIdentity tries to load nonexistant IdentityModel.Net45.dll

### ISSUE THREAD

**Issue by vidarkongsli** [IdentityServer/IdentityServer3.AspNetIdentity#75]
I am setting up IdSrv3 (2.5.4) with IdentityServer3.AspNetIdentity (2.0.0) and IdentityModel (2.0.0 / 1.13.0).
I get this exception in the event log when trying to do a UserService operation:

> System.IO.FileNotFoundException: Could not load file or assembly 'IdentityModel.Net45, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.
> File name: 'IdentityModel.Net45, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'
   at IdentityServer3.AspNetIdentity.AspNetIdentityUserService2.<UpdateAccountFromExternalClaimsAsync>d__29.MoveNext()

Is the newer versions of IdentityModel not compatible?

**Comment by vidarkongsli** [comment]
I downgraded IdentityModel to 1.3.1 and it now works. IdentityModel 1.4.0 and newer breaks.

**Comment by leastprivilege** [comment]
2.0 is a breaking change (by design).

**Comment by vidarkongsli** [comment]
Do you mean IdentityModel 2.0 is a breaking change? Or IdentityServer3.AspNetIdentity 2.0?

**Comment by leastprivilege** [comment]
IdentityModel - it's using semantic versioning - so yes from 1 to 2 there are breaking changes.

**Comment by vidarkongsli** [comment]
Right. However, IdentityModel versions between 1.4 and 1.13 seems to break IdentityServer3.AspNetIdentity. Nuget package dependency states that i IdentityServer3.AspNetIdentity depends on IdentityModel version >= 1.0.0, meaning that 1.4 thru 1.13 should work.

**Comment by sickboy** [comment]
I also ran into this. I switched back and forth between 2.0 and 1.13 a couple of times, im not sure whats up now.

**Comment by sickboy** [comment]
Not sure why a dll's name changed (breaking change), in a minor update (1.13), but it would require IdentityServer3.AspNetIdentity to be updated: https://github.com/IdentityServer/IdentityServer3.AspNetIdentity/issues/76
